src/server.ts

```diff
@@ -33,6 +37,14 @@ export default (
         const request = req.body;
         logger.info('Received a request', { correlationId });
 
+        const cookie = req.header('cookie');
+        const clientMetadata = new Metadata();
+        if (cookie) {
+            logger.debug('cookie', cookie, { correlationId });
```

Comment:
In the [doc](https://concordium.atlassian.net/wiki/spaces/DO/pages/1041203217/Wallets+backend+refactoring) it says that it should be on "trace" level. I guess this translates to `verbose` here?

Reply:
Hmm, I 'translated' trace to debug. :thinking:
(I'd think that verbose < debug < trace)

Reply:
Right, the order is
```
const levels = {
  error: 0,
  warn: 1,
  info: 2,
  http: 3,
  verbose: 4,
  debug: 5,
  silly: 6
};
```

Using `silly` sounds silly so let's just keep it like this ;)